vm/src/builtins.rs

```diff
@@ -139,27 +143,28 @@ fn builtin_compile(args: CompileArgs, vm: &VirtualMachine) -> PyResult {
         .flags
         .map_or(Ok(0), |v| i32::try_from_object(vm, v.into_object()))?;
 
-    if (flags & ast::PY_COMPILE_FLAG_AST_ONLY).is_zero() {
-        #[cfg(feature = "rustpython-compiler")]
-        {
+    #[cfg(feature = "rustpython-parser")]
```

Comment:
Wouldn't `feature = "rustpython-parser"` be guaranteed here, because of the cfg on the whole function?

Reply:
The question here is what happens if someone has a `compiler` but not the `parser`. Not sure if that is possible but maybe the error message is not good.